src/lib/snack-bar/base-snack-bar.ts

```diff
@@ -0,0 +1,12 @@
+import {MdSnackBarRef} from './snack-bar-ref';
+
+
+export class BaseSnackBarContent<T> {
```

Comment:
In the dialog, we just make the `MDDialogRef` available for injection into the component (see `DialogInjector`). You don't have to worry about this for now, though.